### PR TITLE
Update MHKiT-Python installation instructions

### PR DESCRIPTION
**Pull request by simmsa**
Summary of changes:

* Recommends Conda as the preferred installer
* Adds details for installing Conda
* Adds command to download `environment.yml` from MHKiT-Python master
  * `curl -o mhkit_environment.yml https://raw.githubusercontent.com/MHKiT-Software/MHKiT Python/master/environment.yml` 
  * This is a quick option for users to download the most current environment.yml
  * This worked for me using the anaconda terminal on Windows, but there are probably edge cases where this will not work
  * Added a link to the file if using `curl` doesn't work
* Adds install command using conda-forge, fixing #71 
* Changes python test library to `pytest`, fixing #72

### CODE REVIEW

**Comment by simmsa** [comment]
@ssolson and @akeeste do either of you have any comments on this PR?

File: docs/source/installation.rst [hunk 1]
```diff
@@ -21,49 +21,159 @@ Requirements
...
+"""""""""""""""""""""
 
-This option is recommended for MHKiT-Python users.
-To install MHKiT-Python using `pip <https://pip.pypa.io/en/stable/>`_::
+For most users, `Conda <https://docs.conda.io/projects/conda/en/stable/index.html>`_ is the preferred choice for installing MHKiT-Python. Conda includes the ``conda`` command line utility to manage Python versions and dependencies within `environments <https://business-docs.anaconda.com/en/latest/user/environment.html#:~:text=An%20environment%20is%20a%20folder,from%20other%20collections%20of%20packages.>`_. Anaconda provides a standardized Python build environment that is compatible across operating systems, simplifying MHKiT-Python installation and setup for all users.
 
-	pip install mhkit
 
-Using this option to install MHKiT-Python will automatically install Pecos.
+1. `Install Conda <https://docs.conda.io/projects/conda/en/stable/user-guide/install/index.html>`_ with Anaconda Distribution or Miniconda:
```
Comment (akeeste) [inline docs/source/installation.rst:33]: reference the distribution comparison here https://docs.anaconda.com/distro-or-miniconda/

File: docs/source/installation.rst [hunk 2]
```diff
@@ -21,49 +21,159 @@ Requirements
 * `Requests <https://requests.readthedocs.io/>`_: used to get data from websites
 * `Pecos v0.1.9 <https://pecos.readthedocs.io/>`_: used for quality control analysis
 
-
 Install MHKiT-Python
 ^^^^^^^^^^^^^^^^^^^^^
 
-Option 1: PIP Install from Python
-""""""""""""""""""""""""""""""""""""
+Option 1: Using Conda
```
Comment (akeeste) [inline docs/source/installation.rst:27]: I prefer using conda and agree with that as option 1. I think we should further update these steps and discuss git first. This was an issue we ran into at the last training course. When new users come to MHKiT, they very likely want/need the example notebooks. To get those, one needs to use git. An adjustment to these steps could proceed like this:
- Decide whether to use git. 
   - Pros: examples will be installed, enables the developer workflow, users are later one step closer to opening issues and PRs
   - Cons: requires installing / using another software that is not required to run MHKiT
- If yes git:
   - install git for windows 
   - clone repo using the single command line
   - use environment.yml file included in the repo (user workflow) OR pip install the local clone (developer workflow)
- If no git
   - curl to get environment.yml file and conda install OR pip install

File: docs/source/installation.rst [hunk 3]
```diff
@@ -21,49 +21,159 @@ Requirements
...
+- Click the play button next to ``base(root)`` and select "Open Terminal"
+
+.. image:: ./figures/install_anaconda_open_base_environment_terminal.png
+  :width: 500
+  :alt: Opening the terminal for the ``mhkit`` environment
+
+
+Using Miniconda:
+
+- Launch your preferred terminal and execute the following command:
```
Comment (akeeste) [inline docs/source/installation.rst:61]: I don't think conda is guaranteed to be active in any terminal (e.g. command prompt). I think we direct users to search for and open the anaconda terminal directly

**Review by akeeste — COMMENTED** [review]
Thanks @simmsa. My main comment is on how we fit git into the installation workflow, primarily because of how incredibly useful the examples are to new users.
